### ISSUE TITLE
Sounds with spaces in filenames crash loader

### ISSUE THREAD

**Issue by AlexBezuska** [SplatJS/splat-ecs#45]

**Comment by CaldwellYSR** [comment]
Who puts spaces in file names?

**Comment by AlexBezuska** [comment]
Yeah just reporting a discovery I had, I moved a duplicate sound in a folder and it added " 2" (space-2) in the filename and splat threw an error. I know it might be uncommon but it might also be easy to fix and avoid future end users who wouldn't know what's up. (low priority)
